src/pages/Admin/Admin_v2.tsx

```diff
@@ -99,12 +99,7 @@ const Admin_v2: React.FC = () => {
   const renderVenueView = () => {
     switch (selectedOption) {
       case SidebarOptions.dashboard:
-        return (
-          <VenueDetails
-            venue={selectedVenue as Venue_v2}
-            onSave={() => setSelectedOption(sidebarOptions[0].id)}
-          />
```

Comment:
Why was the `onSave` removed here? Should other aspects related to `setSelectedOption` be removed as well? This feels like it might not be related to the stated purpose of this PR?

Reply:
VenueDetails expects only one prop `venue`, that's why I'm removing the `onSave` it's not being used.

Reply:
Makes sense